### ISSUE TITLE
Add glob `ignore` argument, and default to include `node_modules` folder

### ISSUE THREAD

**Issue by marcelolx** [looker-open-source/look-at-me-sideways#56]
We have added LAMS to our project and have found that some reported issues were due to views that don't even exist in our codebase, digging into it I discovered that those views come from the [lookml-parser](https://github.com/fabio-looker/node-lookml-parser) dependency that includes them to be linted with the project  (i.e. https://github.com/fabio-looker/node-lookml-parser/blob/9cf49326ece2bd9834535bf77cec1087554ccbfe/test-projects/002-ndt/ndt.view.lkml) 

![image](https://user-images.githubusercontent.com/20196775/148784095-d98d923f-d9ce-40eb-9eb4-9a516795ce3f.png)

Also when created a new project and LAMS is added it fails to run because of those same views that the parser is including, it raises the following error ⬇️ 

```
TypeError: Cannot read property 'match' of undefined
    at pkNamingConvention (/home/marcelolx/looker/node_modules/@looker/look-at-me-sideways/rules/k1-2-3-4.js:28:23)
    at Array.filter (<anonymous>)
    at module.exports (/home/marcelolx/looker/node_modules/@looker/look-at-me-sideways/rules/k1-2-3-4.js:38:61)
    at module.exports (/home/marcelolx/looker/node_modules/@looker/look-at-me-sideways/index.js:111:17)
    at async /home/marcelolx/looker/node_modules/@looker/look-at-me-sideways/cli.js:26:2
error Command failed with exit code 1.
```
And when checking why it fails I can see that it fails on a dimension from the view that only exists in lookml-parser
```js
{
  '$strings': [
    '@$type',
    ':',
    ' ',
    '@$name',
    ' ',
    '{',
    [ 'hidden', 'hidden', ':', ' ', '@' ],
    '}'
  ],
  '$type': 'dimension',
  '$name': 'user_id',
  hidden: true,
  '$view': 'user_90_day_facts'
}
```

I have created a project if that helps https://github.com/marcelolx/looker
- checkout
- yarn install
- yarn lint

**Comment by fabio-looker** [comment]
Well, this is quite the interesting issue. I don't think I expected anyone to have their LookML project folder also be a Node project with a node_modules folder.

However, looking at your project I assume that this was an unintentional situation since I don't see any other javascript in your folder. It seems you probably omitted the `-g` (global install) flag when installing LAMS, so that's why you ended up with it inside your project folder

**Comment by marcelolx** [comment]
We have it this way because it makes it much easier for people to run the linter locally without having to install it globally (just a `yarn install` & `yarn lint`). 

Couldn't LAMS just ignore the `node_modules` folder from the scan? 🤔

**Comment by fabio-looker** [comment]
Yes, I think so, and I can include that in the next release that I am working on (probably a week or two out from release) Until then, I think you _can_ exclude it explicitly in the `source` argument

**Comment by fabio-looker** [comment]
Implementation note: 

Glob docs: https://www.npmjs.com/package/glob

Argument to be passed through: `ignore`

**Comment by marcelolx** [comment]
Yes, we changed our scripts to only lint the folders from our project. Thanks for the quick responses!

**Comment by fabio-looker** [comment]
Implemented in commit ced9a1af08163a9014f05f3cca3eddaf1f4e8294 and will be included in the next release

**Comment by marcelolx** [comment]
Thank you!

**Comment by fabio-looker** [comment]
This is now published to NPM under v2.1.0

To exclude a folder pass something like --ignore=folder_name/**
